maintainers-guide.md

```diff
@@ -189,36 +189,52 @@ issues.
 
 ## Team labels
 
-* `team-Android`: Issues for Android team ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Android))
-* `team-Apple`: Issues for Apple team (also MacOS, iOS etc) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Apple))
-* `team-Configurability`: Issues for Configurability team ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Configurability))
-* `team-EngProd`: Issues for EngProd team (also release, distribution) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-EngProd))
-* `team-Execution`: Issues for Execution phase team (local and remote execution,
-  sandbox) ([Open
-  issues](https://github.com/bazelbuild/bazel/labels/team-Execution))
+* `team-Android`: Issues for Android team
```

Comment:
Did you consider turning this into a table instead of a list?

```
Label | TL | Description
--- | --- | ---
[`team-Android`](https://github.com/bazelbuild/bazel/labels/team-Android) | @ahumesky | Android-specific issues
...
```

And possibly drop the "Open issues" separate link and just make the label name clickable?

Reply:
This was a table :-) https://github.com/bazelbuild/bazel-website/pull/149/commits/481dcbfbb115469a651217e7a9d20ca64e3cc6c1#r233129420

I'll apply your second suggestion.